web/src/lib/domain-logic/getContentWithCardCount.ts

```diff
@@ -0,0 +1,30 @@
+import { filterCertifications } from "@/lib/domain-logic/filterCertifications";
+import { filterFundings } from "@/lib/domain-logic/filterFundings";
+import { getVisibleCertifications } from "@/lib/domain-logic/getVisibleCertifications";
+import { getVisibleFundings } from "@/lib/domain-logic/getVisibleFundings";
+import { Certification, Funding } from "@/lib/types/types";
+import { LookupOperatingPhaseById } from "@businessnjgovnavigator/shared/operatingPhase";
+import { Business } from "@businessnjgovnavigator/shared/userData";
+
+export const getContentWithCardCount = (
```

Comment:
[pebble] I'm a little confused by this function name... it's `getContent...` but it returns a number. 

I think this could be improved to describe more directly what this function does.
- `getCardCount`
- `getSidebarCardCount`
- `getForYouCardCount`
- `countCards`

Reply:
getForYouCardCount